### ISSUE TITLE
500 Server Error

### ISSUE THREAD

**Issue by FlyingDiver** [FlyingDiver/Indigo-miniPurple#3]
```
  miniPurple Debug                Sensor: getData for sensor 81321:

<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>500 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered an error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

   miniPurple Error                Sensor: getData 'results' key missing: 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>500 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered an error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>
```

**Comment by FlyingDiver** [comment]
API now requires an API Key?

https://api.purpleair.com

Sent email to request Read Key.

**Comment by FlyingDiver** [comment]
https://github.com/FlyingDiver/Indigo-miniPurple/releases/tag/2202.1.0

**Comment by FlyingDiver** [comment]
Fixed in 2022.1.1.
Requires user to obtain an API key.
